modules/openapi-generator/src/main/resources/swift5/modelObject.mustache

```diff
@@ -1,4 +1,4 @@
-{{^objcCompatible}}{{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} {{#useClasses}}final class{{/useClasses}}{{^useClasses}}struct{{/useClasses}} {{classname}}: Codable, Hashable {
+{{^objcCompatible}}{{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} {{#useClasses}}final class{{/useClasses}}{{^useClasses}}struct{{/useClasses}} {{classname}}: Codable{{#vendorExtensions.x-swift-hashable}}, Hashable{{/vendorExtensions.x-swift-hashable}} {
```

Comment:
@maoyama shouldn't we also check here for `{{#hashableModels}}...{{/hashableModels}}`?

Reply:
@4brunu We don't have to check here.
I set x-swift-hashable to true when hashableModels is true to reduce the logic in modelObject.mustache. fb2c45f

Reply:
Nice, good idea! 👍

Reply:
Thanks!